src/components/toolbar/toolbar.scss

```diff
@@ -0,0 +1,19 @@
+@import "@material/toolbar/mdc-toolbar";
+@import "@material/button/mdc-button";
+
+.toolbar {
+  display: inline-flex;
+  flex-direction: column;
+  flex-grow: 1;
+  height: 100%;
+  box-sizing: border-box;
+  &__button {
+    margin-right: 10px;
+    background: none;
```

Comment:
please leave all default ( by material design) styles for the button -- we do not need border bottom (like in mockup)